build-system/tasks/prepend-global/index.js

```diff
@@ -169,6 +169,7 @@ function enableLocalDev(config, target, configJson) {
   const TESTING_HOST = process.env.AMP_TESTING_HOST;
   if (typeof TESTING_HOST == 'string') {
     LOCAL_DEV_AMP_CONFIG = Object.assign(LOCAL_DEV_AMP_CONFIG, {
+      thirdPartyUrl: 'http://' + TESTING_HOST,
```

Comment:
I didn't find `thirdPartyUrl` in production v0.js. is it something still in use? @erwinmombay

Reply:
yes. it is a way for OTHER cdn's to override the 3p url not for ours since ours is the default

Reply:
I see. 

@rsimha-amp I can see why you pointed this value to TESTING_HOST, so that the 3p frame can be loaded from the local server. But one caveat is that the frame will have the same origin as the parent page, which will cause ads behave differently.

Reply:
I see. That might explain some of the errors I saw earlier.

Reply:
I'll let @bradfrizzell comment on whether this caveat is a problem. The good news though, is that we can now see ads on the test page when it's served from a non-localhost server,  so the original issue is fixed.

Reply:
It's surely better than not showing ads on non-localhost servers.

But please make sure this does not affect localhost server. Crazy ads are trying hard to jail break the iframe. If it's on same origin, they will add scripts to host page and make it non-AMP, which might break the testing page.

Reply:
Looking at the code, seems the change also applies to localhost case.  can you confirm?

Reply:
@lannka I don't think this applies to the case where the server is served from `localhost`. If you don't set `AMP_TESTING_HOST`, we do not set `getMode().thirdPartyUrl`, and therefore, default to the old behavior for `localhost`.

Does this make sense?

Reply:
@lannka PTAL at the latest commit. In `getAdsLocalhost()` (called when `localDev` is enabled), we're overriding the value of the default 3p ad frame if the developer has explicitly set `AMP_TESTING_HOST`.

@bradfrizzell has confirmed that this fixes the case where ads on the test page were 404ing for a non-localhost server. This does not affect production, or normal testing with a `localhost` server.